File: 1-TwoSum/1-TwoSum.py
# Last updated: 9/13/2025, 1:37:06 PM
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        ln=len(nums)-1
        for i in range(ln):
            for j in range(i+1,ln+1):
                if nums[i]+nums[j]==target:
                    return [i,j]
        

File: 1-TwoSum/1-TwoSum.cpp
// Last updated: 4/4/2026, 12:43:00 AM
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        vector <int> indices;
        bool found=false;
        int sz=nums.size();
        for(int i=0;i<sz-1;i++){
            for (int j=i+1;j<sz;j++){
                if((nums[i]+nums[j]) ==target){
                    indices.push_back(i);
                    indices.push_back(j);
                    found=true;
                    break;
                }
            
            }

            if (found==true) break;

        }
        return indices;
    }
};